adapters/repos/db/shard_async_replication.go

```diff
@@ -91,9 +91,12 @@ type asyncReplicationConfig struct {
 	propagationDelay            time.Duration
 	propagationConcurrency      int
 	propagationBatchSize        int
+	targetNodeOverrides         []models.AsyncReplicationConfigTargetNodeOverridesItems0
```

Comment:
would use an internal struct, leaving the models for the API

Reply:
good idea, will do, thanks